### ISSUE TITLE
Styles not applied in a CSS-Modules project

### ISSUE THREAD

**Issue by gargantuan** [tuchk4/storybook-readme#15]
My project uses CSS Modules. Following the Readme instructions, I have a webpack.config like this...

```
module.exports = {
  module: {
    loaders: [
      {
        test: /\.md$/,
        loader: 'raw'
      }, {
        test: /\.json$/,
        loader: 'json'
      },
      {
        test: /\.css$/,
        exclude: 'storybook-readme',
        loader: 'style!css?modules&localIdentName=[name]__[local]___[hash:base64:5]',
      }
    ]
  }
};

```

But the outputted Readme text renders un-styled. The formatting is correct, but the styles are not applied.

**Comment by tuchk4** [comment]
In nearest days I will update addon and there will be no  webpack additional configurations

**Comment by gargantuan** [comment]
Cool, that's fixed it. Looking forward to the update that doesn't need webpack config. 

Here's the full working webpack config for anyone who might need it. 

```
module.exports = {
  module: {
    loaders: [
      {
        test: /\.md$/,
        loader: 'raw'
      }, {
        test: /\.json$/,
        loader: 'json'
      },
      {
        test: /highlight\.js\/styles.+\.css$/,
        loader: 'style!css',
      },
      {
        test: /github-markdown-css\/.+\.css$/,
        loader: 'style!css',
      },
      {
        test: /\.css$/,
        exclude: [
          /highlight\.js\/styles.+\.css$/,
          /github-markdown-css\/.+\.css$/,
        ],
        loader: 'style!css?modules&localIdentName=[name]__[local]___[hash:base64:5]',
      }
    ]
  }
};
```

**Comment by tuchk4** [comment]
I have published `storybook-readme@2.0.0` without additional webpack configuration. Only for `*.md` files

```js
module.exports = {
  module: {
    loaders: [{
      test: /\.md$/,
      loader: "raw"
    }]
  }
};
```

**Comment by gargantuan** [comment]
Cool. I'll update later today and let you know if I run into anything.
